packages/dialect-react-ui/components/Chat/screens/Main/ThreadsList/index.tsx

```diff
@@ -22,8 +23,8 @@ const ThreadsList = ({ onThreadClick }: ThreadsListProps) => {
   const {
     params: { threadId },
   } = useRoute<{ threadId?: ThreadId }>();
-  const { threads } = useThreads();
-  // const threads = [];
+  const { pollingInterval } = useChatInternal();
+  const { threads } = useThreads({ refreshInterval: pollingInterval });
```

Comment:
IMO, the polling interval could live in `dialectConfig`, so you could set the default value once and it's will be used without props-drilling and inside `react-sdk`, not `react-ui`. Wdyt?

Reply:
As discussed offline, polling is more related to UI, rather than the SDK. We don't want it to be global for all SWR hooks. 

Currently the only prop-drilling that's happening is from the very high level components to the the `ChatProvider`. After that we access the context in necessary components and use it granularly.

So for now, leaving as is.